source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -0,0 +1,115 @@
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can remove documents by passing a query filter in the
+:java-docs:`deleteOne() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteOne(org.bson.conversions.Bson)>`, 
+:java-docs:`deleteMany() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteMany(org.bson.conversions.Bson)>` 
+or :java-docs:`findOneAndDelete() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#findOneAndDelete(org.bson.conversions.Bson)>` methods.
+
+The ``deleteOne()`` method deletes a single document. If the query
+filter matches more than one document, the method will remove the first
+occurrence of a match in the collection. 
+
+The ``deleteMany()`` method deletes all documents that match the query
+filter.
+
+The ``findOneAndDelete()`` method deletes the first occurrence of a
+match in the collection. 
+
+Examples
+--------
+
+The following examples are about a paint store that sells eight different
+colors of paint. The store had their annual online sale resulting in the
+following documents in their ``paint_inventory`` collection: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "red", "qty": 5 }
+    { "_id": 2, "color": "purple", "qty": 8 }
+    { "_id": 3, "color": "blue", "qty": 0 }
+    { "_id": 4, "color": "white", "qty": 0 }
+    { "_id": 5, "color": "yellow", "qty": 6 }
+    { "_id": 6, "color": "pink", "qty": 0 }
+    { "_id": 7, "color": "green", "qty": 0 }
+    { "_id": 8, "color": "black", "qty": 8 }
+
+``deleteMany()``:
+`````````````````
+
+The paint store website displays all documents in the
+``paint_inventory`` collection. To reduce customer confusion, the store
+wants to remove the colors that are out of stock (``qty`` is ``0``).
```

Comment:
```suggestion
wants to remove the colors that are out of stock.
```
I don't think this is necessary here, especially since it is detailed immediately in the next paragraph.